src/jit/error.cpp

```diff
@@ -363,10 +363,15 @@ void logf_stdout(const char* fmt, va_list args)
         wchar_t wbuffer[BUFF_SIZE];
 
         // Convert char* 'buffer' to a wchar_t* string.
+#ifdef PLATFORM_UNIX
+        WszMultiByteToWideChar(CP_ACP, 0, &buffer[0], -1, &wbuffer[0], BUFF_SIZE);
```

Comment:
We can just use the #else part for Unix (it matches with the ifdefed out _setmode in crossgen main)